### PR TITLE
Update rebuild amount

### PR DESCRIPTION
**Pull request by trofi**
###### Motivation for this change

Improve default help message to have a good executable example.

###### Things done

- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-burns** [comment]
Looks like ofborg is complaining about the tabs in this file. Would you mind either converting to spaces, or adding an editorconfig entry for it with `indent_style = tab`?

**Comment by trofi** [comment]
Sounds good! Added a separate commit to convert tabs to 4-spaces (had to do a minor tweak around here-document to keep it working).

File: maintainers/scripts/rebuild-amount.sh [hunk 1]
```diff
@@ -4,123 +4,130 @@ set -e
...
+    exit 1
 fi
 
 # A slightly hacky way to get the config.
 parallel="$(echo 'config.rebuild-amount.parallel or false' | nix-repl . 2>/dev/null \
-			| grep -v '^\(nix-repl.*\)\?$' | tail -n 1 || true)"
+            | grep -v '^\(nix-repl.*\)\?$' | tail -n 1 || true)"
 
-echo "Estimating rebuild amount by counting changed Hydra jobs."
+echo "Estimating rebuild amount by counting changed Hydra jobs (parallel=$parallel)."
```
Comment (r-burns) [inline maintainers/scripts/rebuild-amount.sh:31]:
```suggestion
echo "Estimating rebuild amount by counting changed Hydra jobs (parallel=${parallel:-unset})."
```
I think this is a bit more descriptive when parallel is not set in the config, than `(parallel=)`

**Review by r-burns — APPROVED** [review]
Looking good, thanks!

**Pull request merged**